ginkgo.Jenkinsfile

```diff
@@ -53,6 +53,24 @@ pipeline {
                }
             }
         }
+        stage('UnitTesting') {
```

Comment:
Rename this stage to `Privileged Unit Testing` and add a comment in the `Jenkinsfile` indicating why we are running this stage in Jenkins vs. in Travis.

Reply:
Does this file just accept `// comments beginning with slashes` anywhere in the file?

Reply:
Yes, `//` indicates a comment in Groovy.